app/src/main/java/vn/realtest/stock/justanewproject/Fragments/FavoriteFragment.java

```diff
@@ -41,25 +46,30 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         view = inflater.inflate(R.layout.fragment_favorite, container, false);
         marketStockList = new ArrayList<>();
         rv_favorite = (RecyclerView) view.findViewById(R.id.rv_favorite);
-        mAdapter = new MarketAdapter(marketStockList);
+        mAdapter = new HoseAdapter(marketStockList);
         RecyclerView.LayoutManager mLayoutManager = new LinearLayoutManager(getContext());
         rv_favorite.setLayoutManager(mLayoutManager);
         rv_favorite.setAdapter(mAdapter);
         prepareStockData();
+        receiveData();
```

Comment:
I prefer getting all favorite stocks from the database. Whenever the user press and hold on the item on the list, it'll be saved to the database and we can get all favorite stocks from that

Reply:
Oh sure 👍  great idea :D you're getting better day by day 👍